Ch travis integration #162633491

NabunyaLilian opened the pull request:
#### What does this PR do?
 Adds Travis Integration to Authors-Haven project

#### Description of Task to be completed?
* Add a .travis.yml file
* Add a Travis badge to the readme file

#### How should this be manually tested?
After adding tests to the project, travis will automatically run the commit to make a build.

#### What are the relevant pivotal tracker stories ?
[#162633491] (https://www.pivotaltracker.com/n/projects/2231917/stories/162633491)

Commit:
[#162633491]create .travis.yml file

Commit:
chore(Travis Integration) Update .travis.yml file

Ensure that .travis.yml file has all the requirements needed to
integrate with travis.

[Delivers #162633491]

Commit:
chore(Travis integration) Update readme with travis badge

Add the travis badge to the readme file such that travis can be accessed
from the badge.

[Delivers#162633491]

Qadriyah commented:
Good work done, the travis.yml file looks good but you did not install the dependences. You need to install coveralls and coverage and then update the requirements.txt file

NabunyaLilian commented:
Thanks for the feedback, let me rectify that

Commit:
chore(travis integration) Update requirements.txt

update requirements.txt to include all the dependencies in .travis.yml

[maintains #162633491]

Commit:
chore(Travis integration) Update travis file

update travis file to follow conventions

[maintains #162633491]

lym commented:
LGTM
